### ISSUE TITLE
Too many repeats in kineto_traces for pt2 compiled ops

### ISSUE THREAD

**Issue by FindHao** [pytorch-labs/tritonbench#86]
pt2 compiled version:
<img width="1854" alt="image" src="https://github.com/user-attachments/assets/74f7b047-4811-487e-9395-3e837d734a91">
liger version:
<img width="1439" alt="image" src="https://github.com/user-attachments/assets/b56d38e0-c4ce-4b73-ac6d-3ef56d98ed63">
There are many extra repeats in kineto_trace for pt2 compiled version.

**Comment by xuzhao9** [comment]
Are they from the warmup iterations? https://github.com/pytorch-labs/tritonbench/blob/main/docs/kineto_trace.md

**Comment by FindHao** [comment]
> Are they from the warmup iterations? https://github.com/pytorch-labs/tritonbench/blob/main/docs/kineto_trace.md

yes. So the warmup section in kineto traces are expected? but liger's kineto trace doesn't have such warmup section.

**Comment by xuzhao9** [comment]
My mental model is that liger's kineto trace should also have warmup section. It is something weird if not.

Can I have the reproduction command line?

**Comment by FindHao** [comment]
> My mental model is that liger's kineto trace should also have warmup section. It is something weird if not.
> 
> Can I have the reproduction command line?
```
python run.py --op kl_div --mode fwd_bwd  --precision fp32 --metrics kineto_trace --csv  --num-inputs 1
```

**Comment by xuzhao9** [comment]
@FindHao This is the liger kernel trace with warmup iteration:

<img width="704" alt="image" src="https://github.com/user-attachments/assets/4ea98369-5b81-4480-aee4-58a2a4588675" />

It shows that when the profile iteration starts, the warm up iterations have finished. Therefore in the kineto trace, it only shows "single iteration", because all previous warmup iterations have finished.

For PT2 and Torch, when the last iteration on CPU launches the kernel, the GPU kernels from previous warm up iterations are still running on GPU, so it looks like "too many repeats for pt2 compiled ops".

Therefore, to me it seems to be expected behavior.

**Comment by xuzhao9** [comment]
Closing as this is expected behavior. Feel free to reopen it if it is still a problem. @FindHao
